options.go

```diff
@@ -18,6 +18,7 @@ func (fn optionFunc[K, V]) apply(opts *options[K, V]) {
 // options holds all available cache configuration options.
 type options[K comparable, V any] struct {
 	capacity              uint64
+	sizeInBytes           uint64
```

Comment:
`totalCost`

Reply:
I have renamed the property as you suggested